space-time.Rmd

```diff
@@ -32,32 +34,40 @@ ggplot(small, aes(eruptions, waiting)) +
 
 For interactive 3d plots, including true 3d surfaces, see RGL, <http://rgl.neoscientists.org/about.shtml>.
 
-## Drawing maps {#maps}
+## Vector maps {#maps}
 \index{Maps!geoms} \index{Data!spatial}
 
-There are four types of map data you might want to visualise: vector boundaries, point metadata, area metadata, and raster images. Typically, assembling these datasets is the most challenging part of drawing maps. Unfortunately ggplot2 can't help you with that part of the analysis, but I'll provide some hints about other R packages that you might want to look at.
+Using maps as a tool for data visualisation can be decomposed into two linked problems: drawing the map itself, and adding metadata representing another information source to that map. These are not always separate problems in practice, but it is often convenient. 
 
-I'll illustrate each of the four types of map data with some maps of Michigan.
+There are three kinds of map data you are most likely to encounter:  this section discusses mapping with vector data, raster data are discussed in Section \@ref(raster), and simple features data are covered by Section \@ref(sf).
 
-### Vector boundaries
+### Drawing vector boundaries
 
-Vector boundaries are defined by a data frame with one row for each "corner" of a geographical region like a country, state, or county. It requires four variables:
+Perhaps the simplest way to define a map is using vector boundaries. For this kind of map, each row in the data frame specifies the location of one "corner" of a geographical region like a country, state, or county. It requires four variables:
```

Comment:
Do you think it would be better to introduce sf first, because that's the type of data you're mostly likely to encounter in practice? And then talk more about the underlying data later?

Reply:
I'm going to try a slight variation: start from the intuitive latitude/longitude representation because that's how most of us novices think about maps, use that to illustrate mapping with `geom_polygon` and then quickly pivot to `geom_sf` etc. From there start to unpack and talk about what `sf` objects are, etc. I'm not sure if it'll work, but let me see where this goes :slightly_smiling_face:

Reply:
I like it! It would also be nice to find an real example of a island with a lake containing a pond to illustrate some of the reasons that the underlying data format is so complex.

Reply:
The Lake Superior polygon is one of my favourites, the outer is a *lake* and it has islands cut as holes out of it, it really breaks the terminology in a fun way

https://github.com/hypertidy/polyggon/blob/master/figure/README-unnamed-chunk-17-1.png

I'll find a pond on an island in a lake somewhere, but I think that one at least has lakes (that are filled) in the islands (that are holes) in the main lake (which is filled).